token/program-2022/src/extension/confidential_transfer/processor.rs

```diff
@@ -431,7 +432,7 @@ fn process_withdraw(
             .base
             .amount
             .checked_add(amount)
-            .ok_or(TokenError::Overflow)?;
+            .ok_or(ProgramError::InvalidInstructionData)?;
```

Comment:
Same q here, is there a reason why `TokenError::Overflow` shouldn't be returned here?

Reply:
Yes, same mistake here. There were places in the code where the `TokenError::Overflow` error was returned for ElGamal arithmetic, which should have been `ProgramError::InvalidInstructionData`. It appears that I made the mistake of changing it in some wrong places.